pycloudlib/openstack/cloud.py

```diff
@@ -0,0 +1,159 @@
+""" Openstack cloud type."""
+import base64
+from operator import itemgetter
+
+import openstack
+
+from pycloudlib.cloud import BaseCloud
+from pycloudlib.openstack.instance import OpenstackInstance
+
+
+class Openstack(BaseCloud):
+    """ Openstack cloud class."""
+
+    _type = 'openstack'
+
+    def __init__(self, tag, network, timestamp_suffix=True):
```

Comment:
I think we have no choice but to require a network parameter. Pycloudlib can't auto-determine the network to use because there could be many defined for any particular openstack deployment and randomly choosing one seems like a bad idea.